cas/cas.go

```diff
@@ -192,6 +192,9 @@ func (ds Store) WriteACI(r io.Reader) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("error detecting image type: %v", err)
 	}
+
+	// Done peeking, bump the buffer size to something more appropriate for bulk copying
+	br = bufio.NewReaderSize(br, 32768)
```

Comment:
@jonboulle I assumed that was intentionally made the size of the peek, there's no undesirable side-effect of a large buffer size on the peek?